src/APICalls.js

```diff
@@ -0,0 +1,19 @@
+const getAllMovies = () => fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies');
+
+export const getSelectedMovie = (id) => fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`);
+
+export const getTrailerKey = (id) => {
+  return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}/videos`)
+    .then(response => response.json())
```

Comment:
Making a note for our future selves that we chose not to invoke `handleErrors` here because if something goes wrong with fetching the trailer, we don't actually care -- we can just not display the Watch Trailer button.